spec/factories/review.rb

```diff
@@ -10,4 +10,25 @@
       new_review_format true
     end
   end
+
+  factory :six_month_review, class: Review do
+    associate_consultant
+    review_type '6-Month'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.